docs/elasticsearch-spec.asciidoc

```diff
@@ -359,64 +359,51 @@ The init container inherits the image of the main container image if one is not
 [id="{p}-update-strategy"]
 === Update strategy
 
-The Elasticsearch cluster configuration can be updated at any time to:
+Users can choose to constrain the operator on the resources it can use when updating using `updateStrategy' like below.
 
-* Add new nodes
-* Remove some nodes
-* Change Elasticsearch configuration
-* Change Pod resources (memory limits, cpu limit, environment variables, etc.)
-
-On any change, ECK reconciles Kubernetes resources towards the desired cluster definition. Changes occur in a rolling fashion: the state of the cluster is continuously monitored, to allow addition of new nodes and removal of deprecated nodes.
-
-[id="{p}-change-budget"]
-==== Change budget
+[source,yaml]
+----
+spec:
+  updateStrategy:
+    changeBudget:
+      maxSurge: 3
+      maxUnavailable: 1
+----
 
-No downtime should be expected when the cluster topology changes. Shards on deprecated nodes are migrated away so the node can be safely removed.
+`maxSurge` refers to the number of Pods that can be scheduled above the total number of Pods in the currently applied spec.
+`maxUnavailable` refers to the number of Pods that can be unavailable out of the total number of Pods in the currently applied spec. Unavailable is defined as Pod being not ready from Kubernetes perspective.
```

Comment:
Possible rewording:

> `maxSurge` refers to the number of Pods that you can schedule in addition to the total number of Pods.
>  `maxUnavailable` refers to the number of Pods that can be unavailable out of the total number of Pods. Pods are defined as unavailable when they are not ready from a Kubernetes perspective.

Not sure if it's really needed to repeat "in the currently applied spec", it's probably intuitive enough as the text refers to the code sample above.

Reply:
I've added it to distinguish between total number of actual (currently running) pods and total number of pods in the spec (not necessarily created yet). I think that without being specific, it's not immediately clear which one is it. WDYT?

Reply:
How about this:

> `maxSurge` refers to the number of Pods that you can schedule in addition to the total number of Pods that are currently running.
> `maxUnavailable` refers to the number of Pods that can be unavailable out of the total number of Pods that are not created yet. Pods are defined as unavailable when they are not ready from a Kubernetes perspective.